src/types.js

```diff
@@ -1,3 +1,5 @@
+// @noflow
```

Comment:
Flow tests for this file were failing, even on master of the fork 😕 

This addition stops flow (a typing library) from trying to type this library.